vtkKNNGraphFilter.cxx

```diff
@@ -116,7 +117,7 @@ int vtkKNNGraphFilter::RequestData(vtkInformation *vtkNotUsed(request),
         
         //set the edge weight to the distance between the points
         double neighborPoint[3];
-        input->GetPoint(neighborID, neighborPoint);
```

Comment:
Why would you use the distance between the normal vectors interpreted as coordinates? Unless the comment on the line above is wrong, what this should be doing is finding the distance between the points - nothing to do with their normals?